File: c-sharp/ex/constructors/heritage/exercises/animal-interface/Gallina.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AnimalInterface
{
  class Gallina : Animal
  {
    int numeroHuevos;
    public Gallina(string sonido) : base(sonido) { }

    public override void Comunicarse()
    {
      Console.WriteLine("Cuando la gallina pone huevos hace " + sonido);
    }
  }
}

File: java/jse-workspace/fundamentos/src/com/pablomonteserin/clasesabstractasinterfaces/claseabstracta/Gallina.java
package com.pablomonteserin.clasesabstractasinterfaces.claseabstracta;

public class Gallina extends Animal {
int numHuevos;
	Gallina(String sonido) {
		super(sonido);
	}

	@Override
	public void comunicarse() {
		System.out.println("La gallina cuando pone huevos hace " + sonido);
	}
	
	
}
